### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by chrzyki**

### CODE REVIEW

File: cldf/forms.csv [hunk 1]
```diff
@@ -2042,8 +2042,8 @@ Sakha-1-58-1,44953,Sakha,1-58,buːrɣa,buːrɣa,b uː r ɣ a,,19,true,^b uː r 
...
-MandarinChinese-1-58-1,32161,MandarinChinese,1-58,bao4feng1yu3,bao4feng1yu3,p au ⁵¹ + f ə ŋ ⁵⁵ + ɥ y ²¹⁴,,22,false,^b ao 4 f eng 1 yu3 $,MandarinChinese,77161647939002505,暴风雨,,5. no evidence for borrowing,0.0,,,,analyzable compound,bao4-feng1-yu3 [burst.out - wind - rain],0.5,,2. Fairly common,,,0.5,,,,Formal,,0. No evidence for calquing,,,,,
-MandarinChinese-1-58-2,32162,MandarinChinese,1-58,kuang2feng1,kuang2feng1,kʰ u a ŋ ³⁵ + f ə ŋ ⁵⁵,,22,false,^k u a ng 2 f eng 1$,MandarinChinese,87272136640928082,狂风,"synonymous: da4feng1 大风 ,storm’, bao4feng1 暴风, feng1bao4 风暴 ,storm’",5. no evidence for borrowing,0.0,,,,analyzable compound,kuang2 - feng1 [wild/angry - wind],0.5,,,,,0.5,,,,,,,,,,,
+MandarinChinese-1-58-1,32161,MandarinChinese,1-58,bao4feng1yu3,bao4feng1yu3,p au ⁵¹ + f ə n k ⁵⁵ + ɥ y ²¹⁴,,22,false,^b ao 4 f en g 1 yu3 $,MandarinChinese,77161647939002505,暴风雨,,5. no evidence for borrowing,0.0,,,,analyzable compound,bao4-feng1-yu3 [burst.out - wind - rain],0.5,,2. Fairly common,,,0.5,,,,Formal,,0. No evidence for calquing,,,,,
```
Comment (FredericBlum) [inline cldf/forms.csv]: How do those changes arise? Are they confirmed to be correct:

**Review by FredericBlum — COMMENTED** [review]
See comment on the nk change. Otherwise good

File: cldf/forms.csv (hunk 1 above)
Comment (chrzyki) [reply cldf/forms.csv]: I assumed them to be a result of the most recent changes to the orthography profile of Mandarin Chinese. However, at that point I thought that the dataset hadn't been re-generated which I now notice it was. Will investigate.

**Review by LinguList — APPROVED** [review]

File: cldf/forms.csv (hunk 1 above)
Comment (chrzyki) [reply cldf/forms.csv]: @LinguList Are there potentially any uncommited changes for the Mandarin Chinese profile to get the mappings as in the previous commit https://github.com/lexibank/wold/commit/fe81aac85cdf541a3f79a3a7e3d8acbe40f73b25? It looks like there were some additional changes to `eng` (as opposed to just `eng$` etc.).

**Pull request merged**